pkg/core/clock.go

```diff
@@ -78,18 +88,17 @@ func (opt *ClockOption) PreProcess() error {
 		log.Warn("Time will be broken when nanosecond delta is too large or too small")
 	}
 
-	if opt.CheckPidExist {
-		// Since os.FindProcess in unix systems will always succeed
-		// regardless of whether the process exists (https://pkg.go.dev/os#FindProcess),
-		// we need to use process.Signal to check if pid is accessible.
-		process, err := os.FindProcess(opt.Pid)
+	// Since os.FindProcess in unix systems will always succeed
+	// regardless of whether the process exists (https://pkg.go.dev/os#FindProcess),
+	// we need to use process.Signal to check if pid is accessible.
+	process, err := os.FindProcess(opt.Pid)
+	if err != nil {
+		fmt.Printf("Failed to find process: %s\n", err)
```

Comment:
return error too? and how about using log.Error?

Reply:
Fixed.